phpunit.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="tests/config/bootstrap.php">
```

Comment:
Don't need this bootstrap.php file.

Reply:
Yeah, if the tests can be run without calling a bootstrap you can just pull that line out.